libs/dh/admin/feature-user-roles/src/lib/dh-user-roles.component.html

```diff
@@ -15,74 +15,68 @@
 limitations under the License.
 -->
 <ng-container *transloco="let t; read: 'admin.userManagement.tabs.roles'">
-  @if (hasGeneralError()) {
-    <watt-empty-state
-      icon="custom-power"
-      [title]="'shared.error.title' | transloco"
-      [message]="'shared.error.message' | transloco"
-    />
-  } @else if (isLoading()) {
-    <div class="loading-container">
-      <watt-spinner />
-    </div>
-  } @else {
-    <mat-accordion multi class="container">
-      @for (actor of userRolesPerActor(); track actor) {
-        <watt-expandable-card
-          variant="solid"
-          [expanded]="expanded()"
-          *transloco="let translateMarketRole; read: 'marketParticipant.marketRoles'"
-        >
-          @if (!selectMode) {
-            <watt-badge>{{
-              t("assigned.count", { count: (actor.userRoles | filterUserRoles: false).length })
-            }}</watt-badge>
-          }
+  <mat-accordion multi class="container">
+    @for (actor of userRolesPerActor(); track actor) {
+      <watt-expandable-card
+        variant="solid"
+        [expanded]="expanded()"
+        *transloco="let translateMarketRole; read: 'marketParticipant.marketRoles'"
+      >
+        @if (!selectMode) {
+          <watt-badge>{{
+            t("assigned.count", { count: (actor.userRoles | filterUserRoles: false).length })
+          }}</watt-badge>
+        }
 
-          @if (actor.name) {
-            <watt-expandable-card-title
-              >{{ actor.actorNumber }} • {{ actor.organizationName }} – {{ actor.name }} -
-              {{ translateMarketRole(actor.userRoles[0].marketRole) }}
-            </watt-expandable-card-title>
-          } @else {
-            <watt-expandable-card-title
-              >{{ actor.actorNumber }} • {{ actor.organizationName }} -
-              {{ translateMarketRole(actor.userRoles[0].marketRole) }}
-            </watt-expandable-card-title>
-          }
+        @if (actor.name) {
+          <watt-expandable-card-title
+            >{{ actor.glnOrEicNumber }} • {{ actor.organization.name }} – {{ actor.name }} -
+            {{ translateMarketRole(actor.userRoles[0].eicFunction) }}
+            @if (actor.id === user()?.administratedBy?.id) {
```

Comment:
issue: testing the component localy gives an error due to missing mock data. Can we add it?